### PR TITLE
DES-19: Small enhancements

### PR DESCRIPTION
**Pull request by aspencer116**
A few small enhancements. See commit messages for descriptions of changes.

I haven't reviewed in anything but Chrome 😬 forgive me

### CODE REVIEW

File: src/scss/2021/tools/_type.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	font-family: var(--font-mono);
 	font-size: var(--type-size-body-lg);
 	font-weight: weight(light);
-	line-height: 1.4;
+	line-height: 1.5rem;
```
Comment (zastrow) [inline src/scss/2021/tools/_type.scss:37]: `line-height` ideally should not have units, it’s based on a relative ratio to the font-size.

File: src/scss/2021/components/_bar-chart.scss [hunk 2]
```diff
@@ -46,13 +46,13 @@
...
   }
-  
+
   &__value {
     display: inline;
     @include type-mono-micro();
-    line-height: 1;
-    padding: 0.125rem;
-    margin-bottom: 0.5rem;
+    line-height: 1rem;
```
Comment (zastrow) [inline src/scss/2021/components/_bar-chart.scss:53]: `line-height` ideally should not have units, it’s based on a relative ratio to the font-size.

File: src/components/2021/pie-chart.js [hunk 3]
```diff
@@ -15,7 +15,9 @@ const PieChart = (props) => {
       <tr className="cmp-pie-chart__key">
         <th className="cmp-pie-chart__key-name">
           <span className={`cmp-pie-chart__key-marker util-fill-style-${i + 1}`} />
-          {props.dataPoints[i][0]}
+          <span className="cmp-pie-chart__key-text">
```
Comment (zastrow) [inline src/components/2021/pie-chart.js:18]: There is not an associated class with this name in the Sass. Did you mean to make a class?

**Review by zastrow — CHANGES_REQUESTED** [review]
Nothing visually stand out to me when I look this in browser. Just a couple of house keeping things:
1. There was a span element added with a class name that is not in the Sass. If no additional styles are need, it can be left as just a span tag without a class.
2. The `line-height` property value should be unitless unless there is a specific reason to require a unit. If there is a particular reason you feel there should be a unit on that property, please leave a comment explaining the purpose.

File: src/components/2021/pie-chart.js (hunk 3 above)
Comment (aspencer116) [reply src/components/2021/pie-chart.js:18]: Oh yeah, I thought I would need that but didn't.

**Pull request merged**